lib/content_based.py

```diff
@@ -67,3 +61,44 @@ def get_document_topic_distribution(self):
         :rtype: ndarray
         """
         return self.document_distribution
+
+    def get_predictions(self):
+        """
+        Get the expected ratings between users and items.
+
+        :returns: A matrix of users X documents
+        :rtype: float[][]
+        """
+        VT = self.document_distribution
+        V = self.document_distribution.T
+        # mean_ratings = (self.ratings.T / numpy.sum(self.ratings, axis=1)).T
+        # normalized_ratings = self.ratings - mean_ratings
+        # predicted_ratings = normalized_ratings.dot(VT).dot(V) / numpy.sum(VT.dot(V), axis=1) + mean_ratings
+        # predicted_ratings = self.ratings.dot(VT).dot(V) / numpy.sum(VT.dot(V), axis=1)
```

Comment:
The comments here are not needed. Also it would be nice if you describe what is being calculated in the documentation or inline